src/main/java/edu/illinois/cs/dt/tools/minimizer/MinimizerPlugin.java

```diff
@@ -48,22 +51,34 @@ public MinimizerPlugin(final InstrumentingSmartRunner runner) {
         this.builder = new TestMinimizerBuilder(runner);
     }
 
-    private Stream<TestMinimizer> fromDtList(final Path path, MavenProject project) {
-        TestPluginPlugin.info("Creating minimizers for file: " + path);
+    private Stream<TestMinimizer> fromDtList(Path path, MavenProject project) {
+        if (FLAKY_LIST != null) {
+            path = Paths.get(FLAKY_LIST);
+            TestPluginPlugin.info("dt.minimizer.flaky.list argument specified: " + FLAKY_LIST);
+        }
+        TestPluginPlugin.info("Creating minimizers for file: " + path);            
```

Comment:
There's some huge whitespace here at the end of this line that should be removed.

Reply:
Done.